src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -17,7 +17,7 @@
             </li>
           </ul>
         </div>
-        <h5 class="detail-id pull-left" id="wi-detail-id">{{workItem.id}}</h5>
+        <h5 *ngIf="!addNewWI" class="detail-id pull-left" id="wi-detail-id">{{workItem.id}}</h5>
```

Comment:
Can we just do it like if workItem.id available or not? So maybe replace {{workItem.id}} with
{{workItem?.id}} and remove *ngIf="!addNewWI".